### DISCUSSION TITLE
Directus White Label / Custom domain

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15432] [Archive]
Hi,

does it exist White Label / Custom domain option by Directus?

Best regards,
Tony

**Comment by aidenfoxx** [comment]
Sure. Directus supports self-hosting so you can just deploy to whatever host you prefer. 👍

**Reply by rynskiy** [reply]
Yes, but White Label means not just hosting,,,
![white label](https://user-images.githubusercontent.com/54798086/188696613-0b5e90ae-fca4-45fd-8db8-30f2d421883e.PNG)

**Reply by aidenfoxx** [reply]
Aha, I see what you mean. I'd get in touch with the team via chat on their website. They might be able to set you up with something special. The alternative is to roll your own, since the source is open.
